cms/envs/aws.py

```diff
@@ -81,6 +81,9 @@
 CELERY_DEFAULT_QUEUE = DEFAULT_PRIORITY_QUEUE
 CELERY_DEFAULT_ROUTING_KEY = DEFAULT_PRIORITY_QUEUE
 
+# Queue to use for updating grades due to grading policy change
+POLICY_CHANGE_GRADES_ROUTING_KEY = LOW_PRIORITY_QUEUE
```

Comment:
Everything you said makes sense, just want to call out a possible inconsistency.

> and we'll ensure the queue exists on lms when the substitution happens

`<service_variant>/celery.py` defines tasks that are queued in `<service variant>` and run somewhere else. So `cms/celery.py` contains things that will run on `lms` queues, and `lms/celery.py` is currently empty because nothing needs to be queued there and run elsewhere (tasks queued from the LMS will natively run on the correct service variant's queue without any tricks). Also FWIW the text-substitution process does not do anything to "ensue the queue exists"; celery allows you to define anything for the queue and we trust the queues defined in `celery.py` to be there.